### DISCUSSION TITLE
Dependabot bug or other issue: Malformed version number string

### DISCUSSION THREAD

**Discussion by lreed-mdsol** [community/community#120281] [Code Security]
### Select Topic Area

Bug

### Body

I crossed posted this in case it is related:
https://github.com/dependabot/dependabot-core/issues/8490

I think this looks like a Bug in how dependabot parses Docker files / images / names / tags.
OR maybe the source is non- conformant? 

Suspected important part of error

> proxy | 2024/04/19 13:23:42 [016] 200 https://registry.hub.docker.com:443/v2/sbtscala/scala-sbt/tags/list
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> Error processing sbtscala/scala-sbt (ArgumentError)
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> Malformed version number string 7_1.9.9_3.4.1

Not sure if this might be caused by some sbt scala matching code, but I was under the impression this was not working yet (as per https://github.com/dependabot/dependabot-core/issues/352) 

The issue seems to be when attempting to parse the docker image 
**sbtscala/scala-sbt:eclipse-temurin-jammy-17.0.10_7_1.9.9_3.4.1**

The Full error log:
```
updater | 2024/04/19 13:23:40 INFO <job_817148696> Starting job processing
updater | 2024/04/19 13:23:41 INFO <job_817148696> Starting update job for mdsol/sensorcloud-archon-consumer
updater | 2024/04/19 13:23:41 INFO <job_817148696> Checking all dependencies for version updates...
updater | 2024/04/19 13:23:41 INFO <job_817148696> Checking if sbtscala/scala-sbt eclipse-temurin-jammy-17.0.10_7_1.9.9_3.4.1 needs updating
  proxy | 2024/04/19 13:23:41 [012] GET https://registry.hub.docker.com:443/v2/sbtscala/scala-sbt/tags/list
  proxy | 2024/04/19 13:23:41 [012] 401 https://registry.hub.docker.com:443/v2/sbtscala/scala-sbt/tags/list
  proxy | 2024/04/19 13:23:41 [012] Remote response: {"errors":[{"code":"UNAUTHORIZED","message":"authentication required","detail":[{"Type":"repository","Class":"","Name":"sbtscala/scala-sbt","Action":"pull"}]}]}
  proxy | 2024/04/19 13:23:41 [014] GET https://auth.docker.io:443/token?service=registry.docker.io&scope=repository%3Asbtscala%2Fscala-sbt%3Apull&account
  proxy | 2024/04/19 13:23:41 [014] 200 https://auth.docker.io:443/token?service=registry.docker.io&scope=repository%3Asbtscala%2Fscala-sbt%3Apull&account
  proxy | 2024/04/19 13:23:41 [016] GET https://registry.hub.docker.com:443/v2/sbtscala/scala-sbt/tags/list
  proxy | 2024/04/19 13:23:42 [016] 200 https://registry.hub.docker.com:443/v2/sbtscala/scala-sbt/tags/list
updater | 2024/04/19 13:23:42 ERROR <job_817148696> Error processing sbtscala/scala-sbt (ArgumentError)
updater | 2024/04/19 13:23:42 ERROR <job_817148696> Malformed version number string 7_1.9.9_3.4.1
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /usr/local/lib/ruby/3.1.0/rubygems/version.rb:223:in `initialize'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/common/lib/dependabot/version.rb:19:in `initialize'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation_2_7.rb:1547:in `bind_call'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation_2_7.rb:1547:in `block in create_validator_procedure_medium1'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /usr/local/lib/ruby/3.1.0/rubygems/version.rb:204:in `new'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /usr/local/lib/ruby/3.1.0/rubygems/version.rb:204:in `new'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/common/lib/dependabot/version.rb:24:in `new'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation_2_7.rb:968:in `bind_call'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation_2_7.rb:968:in `block in create_validator_method_medium1'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/docker/lib/dependabot/docker/version.rb:29:in `initialize'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /usr/local/lib/ruby/3.1.0/rubygems/version.rb:204:in `new'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /usr/local/lib/ruby/3.1.0/rubygems/version.rb:204:in `new'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/common/lib/dependabot/version.rb:24:in `new'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation.rb:272:in `bind_call'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation.rb:272:in `validate_call'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/_methods.rb:272:in `block in _on_method_added'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/docker/lib/dependabot/docker/update_checker.rb:273:in `comparable_version_from'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/docker/lib/dependabot/docker/update_checker.rb:156:in `remove_version_downgrades'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/docker/lib/dependabot/docker/update_checker.rb:117:in `fetch_latest_tag'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/docker/lib/dependabot/docker/update_checker.rb:105:in `latest_tag_from'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/docker/lib/dependabot/docker/update_checker.rb:98:in `latest_version_from'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/docker/lib/dependabot/docker/update_checker.rb:19:in `latest_version'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:181:in `all_versions_ignored?'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:84:in `check_and_create_pull_request'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:64:in `check_and_create_pr_with_error_handling'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:39:in `block in perform'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:39:in `each'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:39:in `perform'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/updater.rb:45:in `run'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/update_files_command.rb:44:in `block in perform_job'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/opentelemetry-api-1.2.3/lib/opentelemetry/trace/tracer.rb:37:in `block in in_span'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/opentelemetry-api-1.2.3/lib/opentelemetry/trace.rb:70:in `block in with_span'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/opentelemetry-api-1.2.3/lib/opentelemetry/context.rb:87:in `with_value'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/opentelemetry-api-1.2.3/lib/opentelemetry/trace.rb:70:in `with_span'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/opentelemetry-api-1.2.3/lib/opentelemetry/trace/tracer.rb:37:in `in_span'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/update_files_command.rb:18:in `perform_job'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/base_command.rb:37:in `run'
updater | 2024/04/19 13:23:42 ERROR <job_817148696> bin/update_files.rb:44:in `<main>'
updater | 2024/04/19 13:23:42 INFO <job_817148696> Finished job processing
updater | 2024/04/19 13:23:42 INFO Results:
updater | Dependabot encountered '1' error(s) during execution, please check the logs for more details.
updater | +------------------------------------+
updater | |   Dependencies failed to update    |
updater | +--------------------+---------------+
updater | | sbtscala/scala-sbt | unknown_error |
updater | +--------------------+---------------+
updater | time="2024-04-19T13:23:42Z" level=info msg="task complete" container_id=job-817148696-updater exit_code=0 job_id=817148696 step=updater
Footer
© 2024 GitHub, Inc.
Footer navigation
[Terms](https://docs.github.com/site-policy/github-terms/github-terms-of-service)
[Privacy](https://docs.github.com/site-policy/privacy-policies/github-privacy-statement)
[Security](https://github.com/security)
[Status](https://www.githubstatus.com/)
```

dependabot.yml - simple - Finds the docker file but then gets the malformed error. 
```
version: 2

updates:
  - package-ecosystem: "docker"
    directory: "/"
    schedule:
      interval: "weekly"
    labels:
      - "dependencies"
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "weekly"
    labels:
      - "dependencies"
```

Docker File:

```
FROM sbtscala/scala-sbt:eclipse-temurin-jammy-17.0.10_7_1.9.9_3.4.1 as base

#Update OS to latest
RUN apt-get update && apt-get upgrade -y

RUN mkdir -p /opt/
COPY . /opt/
WORKDIR /opt/
RUN mv build.sbt build.sbt.bck
RUN cat build.sbt.bck| sed -r 's/.*% +Test.*//' > build.sbt
RUN sbt compile
CMD sbt run
```

The path for the Docker should be valid as per
https://hub.docker.com/r/sbtscala/scala-sbt/tags?page=&page_size=&ordering=&name=jammy-17

https://hub.docker.com/layers/sbtscala/scala-sbt/eclipse-temurin-jammy-17.0.10_7_1.9.9_3.4.1/images/sha256-3be99b57685f88870170ae32370a3ad2a448bffff739acbfcdd1ca61271b75d2?context=explore

Dependabot parsing error ??
Thanks for any ideas!

**Comment by lreed-mdsol** [comment]
Also, how do I set the label for Dependabot for this?  
First time post. 
Thanks in advance.

**Comment by MughalAR** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> I crossed posted this in case it is related:
> https://github.com/dependabot/dependabot-core/issues/8490
> 
> I think this looks like a Bug in how dependabot parses Docker files / images / names / tags.
> OR maybe the source is non- conformant? 
> 
> Suspected important part of error
> 
> > proxy | 2024/04/19 13:23:42 [016] 200 https://registry.hub.docker.com:443/v2/sbtscala/scala-sbt/tags/list
> > updater | 2024/04/19 13:23:42 ERROR <job_817148696> Error processing sbtscala/scala-sbt (ArgumentError)
> > updater | 2024/04/19 13:23:42 ERROR <job_817148696> Malformed version number string 7_1.9.9_3.4.1
> 
> Not sure if this might be caused by some sbt scala matching code, but I was under the impression this was not working yet (as per https://github.com/dependabot/dependabot-core/issues/352) 
> 
> The issue seems to be when attempting to parse the docker image 
> **sbtscala/scala-sbt:eclipse-temurin-jammy-17.0.10_7_1.9.9_3.4.1**
> 
> The Full error log:
> ```
> updater | 2024/04/19 13:23:40 INFO <job_817148696> Starting job processing
> updater | 2024/04/19 13:23:41 INFO <job_817148696> Starting update job for mdsol/sensorcloud-archon-consumer
> updater | 2024/04/19 13:23:41 INFO <job_817148696> Checking all dependencies for version updates...
> updater | 2024/04/19 13:23:41 INFO <job_817148696> Checking if sbtscala/scala-sbt eclipse-temurin-jammy-17.0.10_7_1.9.9_3.4.1 needs updating
>   proxy | 2024/04/19 13:23:41 [012] GET https://registry.hub.docker.com:443/v2/sbtscala/scala-sbt/tags/list
>   proxy | 2024/04/19 13:23:41 [012] 401 https://registry.hub.docker.com:443/v2/sbtscala/scala-sbt/tags/list
>   proxy | 2024/04/19 13:23:41 [012] Remote response: {"errors":[{"code":"UNAUTHORIZED","message":"authentication required","detail":[{"Type":"repository","Class":"","Name":"sbtscala/scala-sbt","Action":"pull"}]}]}
>   proxy | 2024/04/19 13:23:41 [014] GET https://auth.docker.io:443/token?service=registry.docker.io&scope=repository%3Asbtscala%2Fscala-sbt%3Apull&account
>   proxy | 2024/04/19 13:23:41 [014] 200 https://auth.docker.io:443/token?service=registry.docker.io&scope=repository%3Asbtscala%2Fscala-sbt%3Apull&account
>   proxy | 2024/04/19 13:23:41 [016] GET https://registry.hub.docker.com:443/v2/sbtscala/scala-sbt/tags/list
>   proxy | 2024/04/19 13:23:42 [016] 200 https://registry.hub.docker.com:443/v2/sbtscala/scala-sbt/tags/list
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> Error processing sbtscala/scala-sbt (ArgumentError)
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> Malformed version number string 7_1.9.9_3.4.1
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /usr/local/lib/ruby/3.1.0/rubygems/version.rb:223:in `initialize'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/common/lib/dependabot/version.rb:19:in `initialize'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation_2_7.rb:1547:in `bind_call'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation_2_7.rb:1547:in `block in create_validator_procedure_medium1'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /usr/local/lib/ruby/3.1.0/rubygems/version.rb:204:in `new'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /usr/local/lib/ruby/3.1.0/rubygems/version.rb:204:in `new'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/common/lib/dependabot/version.rb:24:in `new'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation_2_7.rb:968:in `bind_call'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation_2_7.rb:968:in `block in create_validator_method_medium1'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/docker/lib/dependabot/docker/version.rb:29:in `initialize'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /usr/local/lib/ruby/3.1.0/rubygems/version.rb:204:in `new'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /usr/local/lib/ruby/3.1.0/rubygems/version.rb:204:in `new'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/common/lib/dependabot/version.rb:24:in `new'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation.rb:272:in `bind_call'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/call_validation.rb:272:in `validate_call'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/sorbet-runtime-0.5.11193/lib/types/private/methods/_methods.rb:272:in `block in _on_method_added'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/docker/lib/dependabot/docker/update_checker.rb:273:in `comparable_version_from'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/docker/lib/dependabot/docker/update_checker.rb:156:in `remove_version_downgrades'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/docker/lib/dependabot/docker/update_checker.rb:117:in `fetch_latest_tag'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/docker/lib/dependabot/docker/update_checker.rb:105:in `latest_tag_from'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/docker/lib/dependabot/docker/update_checker.rb:98:in `latest_version_from'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/docker/lib/dependabot/docker/update_checker.rb:19:in `latest_version'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:181:in `all_versions_ignored?'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:84:in `check_and_create_pull_request'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:64:in `check_and_create_pr_with_error_handling'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:39:in `block in perform'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:39:in `each'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:39:in `perform'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/updater.rb:45:in `run'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/update_files_command.rb:44:in `block in perform_job'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/opentelemetry-api-1.2.3/lib/opentelemetry/trace/tracer.rb:37:in `block in in_span'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/opentelemetry-api-1.2.3/lib/opentelemetry/trace.rb:70:in `block in with_span'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/opentelemetry-api-1.2.3/lib/opentelemetry/context.rb:87:in `with_value'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/opentelemetry-api-1.2.3/lib/opentelemetry/trace.rb:70:in `with_span'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/vendor/ruby/3.1.0/gems/opentelemetry-api-1.2.3/lib/opentelemetry/trace/tracer.rb:37:in `in_span'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/update_files_command.rb:18:in `perform_job'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> /home/dependabot/dependabot-updater/lib/dependabot/base_command.rb:37:in `run'
> updater | 2024/04/19 13:23:42 ERROR <job_817148696> bin/update_files.rb:44:in `<main>'
> updater | 2024/04/19 13:23:42 INFO <job_817148696> Finished job processing
> updater | 2024/04/19 13:23:42 INFO Results:
> updater | Dependabot encountered '1' error(s) during execution, please check the logs for more details.
> updater | +------------------------------------+
> updater | |   Dependencies failed to update    |
> updater | +--------------------+---------------+
> updater | | sbtscala/scala-sbt | unknown_error |
> updater | +--------------------+---------------+
> updater | time="2024-04-19T13:23:42Z" level=info msg="task complete" container_id=job-817148696-updater exit_code=0 job_id=817148696 step=updater
> Footer
> © 2024 GitHub, Inc.
> Footer navigation
> [Terms](https://docs.github.com/site-policy/github-terms/github-terms-of-service)
> [Privacy](https://docs.github.com/site-policy/privacy-policies/github-privacy-statement)
> [Security](https://github.com/security)
> [Status](https://www.githubstatus.com/)
> ```
> 
> dependabot.yml - simple - Finds the docker file but then gets the malformed error. 
> ```
> version: 2
> 
> updates:
>   - package-ecosystem: "docker"
>     directory: "/"
>     schedule:
>       interval: "weekly"
>     labels:
>       - "dependencies"
>   - package-ecosystem: "github-actions"
>     directory: "/"
>     schedule:
>       interval: "weekly"
>     labels:
>       - "dependencies"
> ```
> 
> Docker File:
> 
> ```
> FROM sbtscala/scala-sbt:eclipse-temurin-jammy-17.0.10_7_1.9.9_3.4.1 as base
> 
> #Update OS to latest
> RUN apt-get update && apt-get upgrade -y
> 
> RUN mkdir -p /opt/
> COPY . /opt/
> WORKDIR /opt/
> RUN mv build.sbt build.sbt.bck
> RUN cat build.sbt.bck| sed -r 's/.*% +Test.*//' > build.sbt
> RUN sbt compile
> CMD sbt run
> ```
> 
> The path for the Docker should be valid as per
> https://hub.docker.com/r/sbtscala/scala-sbt/tags?page=&page_size=&ordering=&name=jammy-17
> 
> https://hub.docker.com/layers/sbtscala/scala-sbt/eclipse-temurin-jammy-17.0.10_7_1.9.9_3.4.1/images/sha256-3be99b57685f88870170ae32370a3ad2a448bffff739acbfcdd1ca61271b75d2?context=explore
> 
> Dependabot parsing error ??
> Thanks for any ideas!
> 
> 

# - ****github/workflows/sync-upstream.yml
